server/src/services/deleteTask.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const db = require('../db/db');
+
+const deleteTask = async (userID, taskID) => {
+  const task = await db.findTask(taskID);
+  if (!task) {
+    throw new Error("Task with this ID does not exists!")
+  }
+
+  if (task.userID !== userID) {
```

Comment:
Unless the user can delete not his task?

Reply:
-